src/Core/Components/List/FluentAutocomplete.razor

```diff
@@ -79,7 +79,7 @@
                                     Width="12px"
                                     Style="cursor: pointer;"
                                     Slot="end"
-                                    Title="@AccessibilityIconDismiss"
+                                    Title="@(MaximumSelectedOptions == 1 ? string.Format(AccessibilityRemoveItem, GetOptionText(SelectedOption)) : AccessibilityIconDismiss)"
```

Comment:
@dvoituron - When this is a single select (`MaximumSelectedOptions == 1`) then the accessibility text for the `Clear` button becomes the `AccessibilityRemoveItem` text which is something like `Remove {0}`.

Reply:
Why did you change the current Title result: `DismissTitle="@(string.Format(AccessibilityRemoveItem, text))"` ?

Reply:
This was an attempt to improve the "accessibility" of the component. Since you can no longer remove the individual item, I felt it might be good for the 'Clear' button to include the name of the selected option and so I just updated it to use the same text as if you were removing a single item.  I'm happy to change it back if you don't feel like this is a helpful update.

Reply:
It's an option to add additional information in this case. But my biggest accessibility problem is that the component must “read” (with NVDA) all the elements present on the screen. This is still not the case when an item is selected, the user must be able to position himself on the component (focusable) and NVDA must announce something like “<SelectedItem> selected”.

To move forward with this PR (and to be compatible with what already exists), it would probably be simpler to allow the `Multiple=false` parameter, which would activate your functionality. If `Multiple=true`, everything would work as it does now, and Accessibility would be compatbile.

Reply:
What do you think about this idea?

Reply:
I tried this by making the following changes:

![image](https://github.com/user-attachments/assets/3da58e6d-a302-42ae-9e46-0c57bc845af2)

However, the control did not seem to work.  It wouldn't allow me to select anything.  I'm not opposed to this idea, I'm just not sure what else would be required to get it to work.  At the end of the day, I'm simply looking for a way to only allow a single item to be selected and also have the UI styled in such a way that is optimal for a single item.  As it stands, I believe this PR accomplishes that and does it in a way that does not break anything existing and is hopefully acceptable from an accessibility perspective with the latest change I pushed.